src/dsp_tools/fast_xmlupload/process_files.py

```diff
@@ -824,15 +819,18 @@ def double_check_unprocessed_files(
     """
     unprocessed_files_txt_exists = sorted(unprocessed_files) != sorted(all_files)
     if unprocessed_files_txt_exists and not processed_files:
+        logger.error("There is a file 'unprocessed_files.txt', but no file 'processed_files.txt'")
         raise UserError("There is a file 'unprocessed_files.txt', but no file 'processed_files.txt'")
```

Comment:
Maybe add to the message, that this means no files were uploaded.

Reply:
This is part of the checks before the actual work begins. I think the user will realize that no work has been done, because there will be only very few terminal output, and there will be no folders with processed files.